gate-web/src/main/groovy/com/netflix/spinnaker/gate/config/GateConfig.groovy

```diff
@@ -97,6 +100,9 @@ class GateConfig extends RedisHttpSessionConfiguration {
   @Value('${retrofit.logLevel:BASIC}')
   String retrofitLogLevel
 
+  @Value('${services.kayenta.externalhttps:false}')
```

Comment:
I'd suggest making this a parameter to the kayentaService method instead of at the class level (its only used there and keeps it close to where its used).

Reply:
Done.  Will merge, thanks!